template-for-repository/proofs/arpa-test.sh

```diff
@@ -0,0 +1,342 @@
+#!/bin/bash
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# This script performs  data collection for Arpa.
+# Furthermore, it serves as a validation framework for existing CBMC proofs.
```

Comment:
What is the purpose of this script?  Is it to invoke Arpa?  Is it to validate Arpa?  Some of the script appears to be collecting data in order to compare the results of the Makefile with and without Arpa.